recipes-python/pycoral/pycoral_2.0.0.bb

```diff
@@ -15,10 +15,8 @@ require recipes-devtools/bazel-toolchains/bazel-toolchains.inc
 
 PR  = "3"
 
-DEPENDS = " \
-    libedgetpu \
-    python3-tflite-runtime \
-"
+DEPENDS += "libedgetpu-dev"
+RDEPENDS += "python3-tflite-runtime"
```

Comment:
Why `+=`?

Reply:
It's a habit inherited from the PROVIDES (where it makes a difference). But you are right. I'll fix it.